doc/source/configuration/firewall.rst

```diff
@@ -0,0 +1,350 @@
+.. _firewall:
+
+========
+Firewall
+========
+
+StackHPC Kayobe configuration provides a standardised firewalld configuration.
+The configuration uses the :kayobe-doc:`firewall
+<configuration/reference/hosts.html#firewalld>` host configuration
+functionality of Kayobe.
+
+The firewall configuration is provided in
+``etc/kayobe/inventory/group_vars/all/firewall``.
+
+Enabling StackHPC firewalld rules
+=================================
+
+The standardised firewalld configuration is not used by default and must be
+actively opted into. This can be done as follows:
```

Comment:
```suggestion
The standardised firewalld configuration is not enabled by default and must be
actively opted into. This can be done by altering or copying ``etc/kayobe/<group>.yml`` to the chosen ``etc/kayobe/environment/<environment>/``, and making the appropriate following changes:
```

Reply:
```suggestion
The standardised firewalld configuration is not enabled by default and must be
actively opted into. This can be done by altering or, in the case of using a ``kayobe`` environment, copying ``etc/kayobe/<group>.yml`` to the chosen ``etc/kayobe/environment/<environment>/``, and making the appropriate following changes:
```

Seems odd that we reference paths and files in terms of kayobe environments later on, but not for this section, when it seems equally valid to do so.

Reply:
I've updated the line to mention both environments and the base config